Bug 1707601 - Add the doc for instructions on how to getting started (#) (#267)
mozilla/glean.js: docs/getting_started.md

# Getting started
This guide is for developers who want to develop on Glean.js.
## Prerequisites

> Glean.js uses [Node.js](https://nodejs.org/).

Reviewer: Also good to mention the minimum required versions here, https://github.com/mozilla/glean.js/commit/6020080d63d6013c526dc8576e42e5c64bc0329a they were added in this commit.

Follow the [official guide](https://nodejs.dev/) to install it.

## Building Glean.js

> 1. At terminal, navigate the directory where you want the Glean.js repository to live. Then:

Reviewer: ```suggestion
1. On a terminal, navigate to the directory you want the Glean.js repository to live.
```

```bash
git clone https://github.com/mozilla/glean.js
```

> 2. 

Reviewer: Missing text on this bullet point.

```bash
cd glean.js/glean
```
3. Install all the node modules used by Glean.js:
```bash
npm install
```

Upon completing step 3, you may proceed to [running tests](#running-tests). 

> If you want to build certain components, please refer to the `scripts` section of [`package.json`](../glean/package.json) 

Reviewer: I think this paragraph can be ommited :)

for other build commands.

## Running tests

> Once you have [build](#building-gleanjs) Glean.js, run

Reviewer: ```suggestion
Once you have [built](#building-gleanjs) Glean.js, run
```

```bash
npm run test
```
in the `glean` folder.

This runs all the tests in the `glean` folder. 
Alternatively, you may want to run component-specific tests. 

> To do so, substitute `test` in the command above with one of the followings:

Reviewer: Can you add a small explanation on what each of these mean? e.g. `npm run test:core` will execute all tests inside `tests/core` folder and so on.

* `test:core`
* `test:plugins`

> * `test:platform`

Reviewer: Let's also add a tip here for how to run specific tests using `--grep` e.g. `npm run test:core -- --grep "specific test"`.
